src/test/integration/controllers/auth.spec.js

```
@@ -114,4 +114,142 @@ describe('Auth controller', () => {
+
+    it('should handle expired Google token', async () => {
+      OAuth2Client.prototype.verifyIdToken.mockRejectedValue(
+        new Error('Token expired')
+      );
+      
+      const response = await app
+        .post('/auth/google')
+        .send({ idToken: 'expired_token' });
+
+      expect(response.status).toBe(401);
+      expect(response.body.code).toBe('INVALID_GOOGLE_TOKEN');
```

ESLint

[error] Extra semicolon.

(semi)